### DISCUSSION TITLE
/oh-my-posh.exe' is not recognized as a name of a cmdlet

### DISCUSSION THREAD

**Discussion by AndresFPerezG** [JanDeDobbeleer/oh-my-posh#2418] [General]
Hi everyone, I recently install oh-my-posh to powershel and followed https://ohmyposh.dev/docs/installation/windows, then to change my prompt I followed this instructions https://ohmyposh.dev/docs/installation/prompt, but when I execute . $PROFILE, I get this error:

&: The term 'C:/Users/Andr├®s/AppData/Local/Programs/oh-my-posh/bin/oh-my-posh.exe' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
Invoke-Expression: Cannot bind argument to parameter 'Command' because it is an empty string.

Or if I run this: oh-my-posh prompt init pwsh --config ~/tonybaloney.omp.json
then get this: (@(&"C:/Users/Andrés/AppData/Local/Programs/oh-my-posh/bin/oh-my-posh.exe" init pwsh --config="C:\Users\Andrés\tonybaloney.omp.json" --print) -join "`n") | Invoke-Expression

and don't know what it means, can you help me?, thanks.

**Comment by JanDeDobbeleer** [comment]
That's because PowerShell and UTF8 is an issue. You need to force the shell to UTF8 (or for the duration of the initialization of oh-my-posh).

```powershell
$OutputEncoding = [System.Console]::OutputEncoding = [System.Console]::InputEncoding = [System.Text.Encoding]::UTF8
```

**Reply by jpolivra** [reply]
That's right.

My name is brazilian so my user name is "João". If not configured as @JanDeDobbeleer mentioned it will return an error.

**Comment by bigbyrd69** [comment]
Helpp

![image](https://user-images.githubusercontent.com/48597410/195211536-ec0a7ba6-0c6e-4a8e-9180-6085a22463b9.png)

**Reply by JanDeDobbeleer** [reply]
@AndresFPerezG you have to restart the terminal. PATH's are only loaded on start.

**Reply by TranManhCuongQN** [reply]
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/89560029/eab4636a-b85b-42c8-8639-ce67b8ce9d0e)
I have a problem as described above and i hope you can help me

**Comment by Flow-sandyu** [comment]
"More specifically, you need to write the command `$OutputEncoding = [System.Console]::OutputEncoding = [System.Console]::InputEncoding = [System.Text.Encoding]::UTF8`  into the configuration file of PowerShell. You can open the configuration file of PowerShell by running `notepad $PROFILE`, and then write the command into the file."
